source/openapi-admin-v3.yaml

```diff
@@ -7860,23 +7855,51 @@ components:
                     the Trigger.
                 database:
                   type: string
-                  description: The name of a database in the linked data source.
+                  description: |-
+                    The name of a database in the linked data source. If you
+                    omit this parameter, the Source Type changes to
+                    "Deployment," unless you are on a shared tier, in which
+                    case App Services will not let you save the trigger.
                 collection:
                   type: string
                   description: |-
                     The name of a collection in the specified database. The
-                    trigger listens to events from this collection.
+                    trigger listens to events from this collection. If you omit
+                    this parameter, the Source Type changes to "Database."
                 operation_types:
                   type: array
                   minItems: 1
                   items:
                     type: string
-                    enum:
-                      - INSERT
-                      - UPDATE
-                      - REPLACE
-                      - DELETE
-                  description: The type(s) of MongoDB change event that the trigger listens for.
+                  description: |-
+                    The type(s) of MongoDB change event that the trigger
+                    listens for.
+                    
+                    Valid operation types for all triggers include:
+                    
+                    - `"INSERT"`
+                    - `"UPDATE"`
+                    - `"REPLACE"`
+                    - `"DELETE"`
+                    
+                    Additional valid operation types for database and
+                    deployment triggers include: 
+                    
+                    - `"CREATE_COLLECTION"`
+                    - `"MODIFY_COLLECTION"`
+                    - `"RENAME_COLLECTION"`
+                    - `"DROP_COLLECTION"`
+                    - `"SHARD_COLLECTION"`
+                    - `"RESHARD_COLLECTION"`
+                    - `"REFINE_COLLECTION_SHARD_KEY"`
+
+                    Additional valid operation types for deployment triggers
+                    include:
+                    
+                    - `"DROP_DATABASE"`
+
+                    For more information, refer to
+                    [Trigger Configuration](https://www.mongodb.com/docs/atlas/app-services/triggers/database-triggers/#configuration).
```

Comment:
just confirming this should go to the `/database-triggers` page

Reply:
That was the plan, because this is specific to Database Triggers, and there is more info there. There's _also_ info on the config reference page. But I think the main "Triggers" page is more likely to be actively maintained - for better or worse, we tend to forget the config reference pages exist and forget to update them.